### PR TITLE
22.1.0-beta.2 release note

### PR DESCRIPTION
**Pull request by stbof**
https://cockroachlabs.atlassian.net/browse/DOC-1909

### CODE REVIEW

File: _includes/releases/v22.1/v22.1.0-beta.2.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+- [Rangefeed](../v22.1/use-changefeeds.html#enable-rangefeeds) memory budgets have a [cluster setting](../v22.1/cluster-settings.html) `kv.rangefeed.memory_budgets.enabled` that disables memory budgeting for all new feeds. This setting could be used on [CockroachDB Dedicated](../v22.1/choose-a-deployment-option.html#dedicated) clusters to disable budgeting as a mitigation for bugs for example if feeds abort while nodes have sufficient free memory. [#79321][#79321]
+- Rangefeed memory budgets could be disabled on the fly when cluster setting is changed without the need to restart the feed. [#79321][#79321]
+
+<h3 id="v22-1-0-beta-2-db-console-changes">DB Console changes</h3>
+
+- Minor styling changes on [Hot Ranges](../v22.1/ui-hot-ranges-page.html) page to follow the same style as other pages. [#79501][#79501]
+- On the [Statement Details] page, changed the order of tabs to Overview, Explain Plan, Diagnostics and Execution Stats and the Explain Plan tab was changed to Explain Plans (plural). [#79234][#79234]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.0-beta.2.md:72]:
```suggestion
- On the [Statement Details](../v22.1/ui-statements-page.html#statement-details-page) page, changed the order of tabs to Overview, Explain Plan, Diagnostics and Execution Stats and the Explain Plan tab was changed to Explain Plans (plural). [#79234][#79234]
```

File: _includes/releases/v22.1/v22.1.0-beta.2.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+- [Rangefeed](../v22.1/use-changefeeds.html#enable-rangefeeds) memory budgets have a [cluster setting](../v22.1/cluster-settings.html) `kv.rangefeed.memory_budgets.enabled` that disables memory budgeting for all new feeds. This setting could be used on [CockroachDB Dedicated](../v22.1/choose-a-deployment-option.html#dedicated) clusters to disable budgeting as a mitigation for bugs for example if feeds abort while nodes have sufficient free memory. [#79321][#79321]
+- Rangefeed memory budgets could be disabled on the fly when cluster setting is changed without the need to restart the feed. [#79321][#79321]
+
+<h3 id="v22-1-0-beta-2-db-console-changes">DB Console changes</h3>
+
+- Minor styling changes on [Hot Ranges](../v22.1/ui-hot-ranges-page.html) page to follow the same style as other pages. [#79501][#79501]
+- On the [Statement Details](../v22.1/ui-statements-page.html#statement-details-page) page, changed the order of tabs to Overview, Explain Plan, Diagnostics and Execution Stats and the Explain Plan tab was changed to Explain Plans (plural). [#79234][#79234]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.0-beta.2.md:72]:
```suggestion
- On the [Statement Details](../v22.1/ui-statements-page.html#statement-details-page) page, changed the order of tabs to Overview, Explain Plan, Diagnostics, and Execution Stats and changed the Explain Plan tab to Explain Plans (plural). [#79234][#79234]
```

File: _includes/releases/v22.1/v22.1.0-beta.2.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+<h3 id="v22-1-0-beta-2-operational-changes">Operational changes</h3>
+
+- The `kv.allocator.load_based_rebalancing_interval` [cluster setting](../v22.1/cluster-settings.html) now lets operators set the interval at which each store in the cluster will check for load-based lease or replica rebalancing opportunities. [#79073][#79073]
+- [Rangefeed](../v22.1/use-changefeeds.html#enable-rangefeeds) memory budgets have a [cluster setting](../v22.1/cluster-settings.html) `kv.rangefeed.memory_budgets.enabled` that disables memory budgeting for all new feeds. This setting could be used on [CockroachDB Dedicated](../v22.1/choose-a-deployment-option.html#dedicated) clusters to disable budgeting as a mitigation for bugs for example if feeds abort while nodes have sufficient free memory. [#79321][#79321]
+- Rangefeed memory budgets could be disabled on the fly when cluster setting is changed without the need to restart the feed. [#79321][#79321]
+
+<h3 id="v22-1-0-beta-2-db-console-changes">DB Console changes</h3>
+
+- Minor styling changes on [Hot Ranges](../v22.1/ui-hot-ranges-page.html) page to follow the same style as other pages. [#79501][#79501]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.0-beta.2.md:71]:
```suggestion
- Minor styling changes on [**Hot Ranges**](../v22.1/ui-hot-ranges-page.html) page to follow the same style as other pages. [#79501][#79501]
```

File: _includes/releases/v22.1/v22.1.0-beta.2.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+- On the [Statement Details](../v22.1/ui-statements-page.html#statement-details-page) page, changed the order of tabs to Overview, Explain Plan, Diagnostics, and Execution Stats and changed the Explain Plan tab to Explain Plans (plural). [#79234][#79234]
```
Comment (stbof) [inline _includes/releases/v22.1/v22.1.0-beta.2.md:72]:
```suggestion
- On the [**Statement Details**](../v22.1/ui-statements-page.html#statement-details-page) page, changed the order of tabs to Overview, Explain Plan, Diagnostics, and Execution Stats and changed the Explain Plan tab to Explain Plans (plural). [#79234][#79234]
```

**Review by Azhng — COMMENTED** [review]
Looks good with some small nits. I'm not sure what's the docs team review process, should we also ping someone from the docs teams ? 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13549#-:-N-ObJcj5JcuxJHWRMjw:bd618ef)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @Azhng and @stbof)

___
*[_includes/releases/v22.1/v22.1.0-beta.2.md, line 83 at r6](https://reviewable.io/reviews/cockroachdb/docs/13549#-N-OauE0BD40cjzc_DEg:-N-OauE0BD40cjzc_DEh:b26718s) ([raw file](https://github.com/cockroachdb/docs/blob/14311a6b462a2eb681c503631cf887ae30dd7cd8/_includes/releases/v22.1/v22.1.0-beta.2.md#L83)):*
> ```Markdown
> - [`SHOW SCHEMAS FROM <schema>`](../v22.1/show-schemas.html) now includes user-defined schemas. [#79308][#79308]
> - A distributed query that results in an error on the remote node no longer has an incomplete trace. [#79193][#79193]
> - [`IMPORT INTO`](../v22.1/import-into.html) now longer creates duplicate entries with [`UNIQUE`](../v22.1/unique.html) constraints in [`REGIONAL BY ROW` tables](../v22.1/multiregion-overview.html#regional-by-row-tables) and tables utilizing `UNIQUE WITHOUT INDEX` constraints. A new post-`IMPORT` validation step for those tables now fails and rolls back the `IMPORT` in such cases. [#79323][#79323]
> ```

nit: "... no longer creates duplicate entrries ... "
___
*[_includes/releases/v22.1/v22.1.0-beta.2.md, line 85 at r6](https://reviewable.io/reviews/cockroachdb/docs/13549#-N-OaHwR6si9IfggeK2M:-N-OaHwR6si9IfggeK2N:b6qibp1) ([raw file](https://github.com/cockroachdb/docs/blob/14311a6b462a2eb681c503631cf887ae30dd7cd8/_includes/releases/v22.1/v22.1.0-beta.2.md#L85)):*
> ```Markdown
> - [`IMPORT INTO`](../v22.1/import-into.html) now longer creates duplicate entries with [`UNIQUE`](../v22.1/unique.html) constraints in [`REGIONAL BY ROW` tables](../v22.1/multiregion-overview.html#regional-by-row-tables) and tables utilizing `UNIQUE WITHOUT INDEX` constraints. A new post-`IMPORT` validation step for those tables now fails and rolls back the `IMPORT` in such cases. [#79323][#79323]
> - Fixed a bug in IO which could result in [admission control](../v22.1/architecture/admission-control.html) failing to rate limit when traffic was stalled such that no work was admitted, despite the store's being in an unhealthy state. [#79343][#79343]
> - The execution time as reported on [`DISTSQL`](../v22.1/explain-analyze.html#explain-analyze-distsql) diagrams within the statement bundle collected via [`EXPLAIN ANALYZE (DEBUG)`](../v22.1/explain-analyze.html#debug-option) is no longer when the statement encountered an error. [#79373][#79373]
> ```

nit: "... is no longer ...", feels something is missing here

**Review by Azhng — COMMENTED** [review]
lgtm from me. I'll defer the stamp to the docs reviewer. 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13549#-:-N-TlL_I1PnVOuO158uV:byxqr7v)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @Azhng, @rmloveland, and @stbof)

**Review by rmloveland — APPROVED** [review]
:lgtm: 

> Reviewed 2 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13549#-:-N-UeHLV9aFbpt0U445N:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @Azhng, @rmloveland, and @stbof)

**Pull request merged**